.github/CODEOWNERS

```diff
@@ -18,6 +18,9 @@
 # PRLabel: %Azure.Identity
 /sdk/azidentity/              @chlowell @jhendrixMSFT @rickwinter @Azure/azure-sdk-write-identity
 
+# PRLable: %OpenAI
+/sdk/ai                       @jhendrixMSFT @richardpark-msft
```

Comment:
I believe the convention here is the first entry is the primary owner and all others are secondary/backup owners.

Reply:
Updated.

Also added in EventGrid.